data/graphics.js

```diff
@@ -13592,6 +13592,34 @@ var BattleMoveAnims = {
 			var attacker = args[0];
 			var defender;
 
+			battle.bgElem.animate({
+				position: 'absolute',
+				left: -5,
+				right: 5,
+			}, 75).animate({
+				left: 5,
+				right: -5
+			}, 100).animate({
+				left: -5,
+				right: 5
+			}, 100).animate({
+				left: 5,
+				right: -5
+			}, 100).animate({
+				left: -5,
+				right: 5
+			}, 100).animate({
+				left: 5,
+				right: -5
+			}, 100).animate({
+				left: 0,
+				right: 0,
+			}, 100, function () {
+				battle.bgElem.animate({
+					position: 'absolute'
```

Comment:
...????

First of all, `position` is not a CSS attribute that can be animated. This will actually do exactly nothing (which is good because the `position` attribute is already set to `absolute` so you don't need it to do anything at all.